README.md

```diff
@@ -199,6 +199,8 @@ expect("this string").to start_with("this")
 expect("this string").to end_with("ring")
 expect([1, 2, 3]).to contain_exactly(2, 3, 1)
 expect([1, 2, 3]).to match_array([3, 2, 1])
```

Comment:
maybe add a trailing comment clarifying what this actually does? Similar to other documentation:

https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers

It's kind of confusing that we have both `contain_exactly` and `match_array`, when they do the same thing just with different argument conventions.

WDYT about something like:

```
 expect([1, 2, 3]).to contain_exactly(2, 3, 1) # Order independent match
 expect([1, 2, 3]).to match_array([3, 2, 1])   # Equivalent to above
expect([1,2,3]).to eq([1,2,3]) # Order dependent match can use standard eq matcher
```

We should also consider adding a link to https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers "for complete documentation", maybe just below the `## Built-in Matchers` subhead.

Reply:
I had assumed that the relishapp.com docs were pulled from this README.  If those are stored somewhere else in this repo (a different branch? hidden in some subdir?) then I'd be glad to modify that as well.